### PR TITLE
Add a `debug` input

### PR DESCRIPTION
**Pull request by remoun**
To facilitate debugging script excecution

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 # Copyright (c) 2022 MobileCoin Inc.
 
+[[ "$INPUT_DEBUG" == 'true' ]] && set -x
```
Comment (jgreat) [inline entrypoint.sh:4]: To be honest this scares me.  I know that GHA does a pretty good job of masking secrets output, but its just doing string matching and its not perfect. Setting `-x` will probably lead to secrets exposure.

I'd prefer we set specific debug lines for things we want to output:
```
[ -n "${INPUT_DEBUG}" ] && echo "## Create target dir ${target_dir}"
```

There's a "proper" way to set debug on an action: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging

I'm not exactly sure how that's presented to the github action.

Comment (jgreat) [reply entrypoint.sh:4]: Or make a function like `debug_echo` to make it a bit more dry.

**Review by jgreat — CHANGES_REQUESTED** [review]
I don't believe we can guarantee that `set -x` will be safe in all cases.  I'd prefer to debug specific information.  See inline comment

File: entrypoint.sh (hunk 1 above)
Comment (remoun) [reply entrypoint.sh:4]: Fair enough. I'll add `debug_echo`

**Comment by remoun** [comment]
Adding `debug_echo` turns out to be quite involved. With that said, I (hopefully) only need this debugging for a one-off, so I'll use the image built for this PR's CI

**Pull request closed without merging**